### PR TITLE
docs: initial version of `COMPONENT-OWNERS.md` file

### LINKED ISSUE #3: Draft initial status of `COMPONENT-OWNERS.md` file
Make an inventory of the existing projects and start assigning the owners to each of those.

### PR DESCRIPTION
**Pull request by gbartolini**
Closes #3

### CODE REVIEW

File: COMPONENT-OWNERS.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# CloudNativePG Component Owners
+
+## charts
+
+**Repository:** [cloudnative-pg/charts](https://github.com/cloudnative-pg/charts)
```
Comment (phisco) [inline COMPONENT-OWNERS.md:5]: I'd suggest splitting the two charts into two separate components 🙏

Comment (sxd) [reply COMPONENT-OWNERS.md:5]: separate components or separate repos? by components you mean like add a CODEOWNERS files in the top of the project and split them from there ?

Comment (gbartolini) [reply COMPONENT-OWNERS.md:5]: What about now?

Comment (sxd) [reply COMPONENT-OWNERS.md:5]: Nice! I think it looks better and more clear now, what do you think @phisco ?

**Comment by gbartolini** [comment]
I am seeking help from maintainers to review all the components I have added, and the people I have tentatively put in the list (based on my understanding). Please make sure you add people you think should be there.

**Review by sxd — APPROVED** [review]

**Review by leonardoce — APPROVED** [review]

**Review by phisco — APPROVED** [review]
🎉 thanks @gbartolini

**Review by mnencia — APPROVED** [review]

**Review by armru — APPROVED** [review]

**Pull request merged**